<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPanel</class>
 <widget class="QWidget" name="MenuPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>52</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>52</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QPushButton" name="helpButton">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>42</height>
      </size>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/resources/icons/sys_icon_help.png</normaloff>:/icons/resources/icons/sys_icon_help.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="modeButton">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>42</height>
      </size>
     </property>
     <property name="text">
      <string>Mode</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/resources/icons/basic_webpage_multiple.png</normaloff>:/icons/resources/icons/basic_webpage_multiple.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="calibrationButton">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>42</height>
      </size>
     </property>
     <property name="text">
      <string>Calibration</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/resources/icons/basic_pencil_ruler.png</normaloff>:/icons/resources/icons/basic_pencil_ruler.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="registrationButton">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>42</height>
      </size>
     </property>
     <property name="text">
      <string>Registration</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/resources/icons/basic_todolist_pencil.png</normaloff>:/icons/resources/icons/basic_todolist_pencil.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="memoryButton">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>42</height>
      </size>
     </property>
     <property name="text">
      <string>Memory</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/resources/icons/sys_icon_microchip.png</normaloff>:/icons/resources/icons/sys_icon_microchip.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="optionsButton">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>42</height>
      </size>
     </property>
     <property name="text">
      <string>Options</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/resources/icons/sys_icon_options.png</normaloff>:/icons/resources/icons/sys_icon_options.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="systemButton">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>42</height>
      </size>
     </property>
     <property name="text">
      <string>System</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/icons/resources/icons/sys_icon_settings.png</normaloff>:/icons/resources/icons/sys_icon_settings.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
